### PR TITLE
Improve recent jobs list on the Header

### PR DESCRIPTION
**Pull request by syucream**
Simplify recent jobs list on the Header, just to show job details w/o any link  (but on only export jobs, it contains a download link).
It also contains these improvements:
- Support `created_after` query on the Jobs API V2 to replace `/api/v1/job/`.
- Check/update the latest date when user checks the recent jobs list to manage badge count.

image

### CODE REVIEW

File: frontend/src/utils/AironeAPIClient.ts [hunk 1]
```diff
@@ -278,10 +278,6 @@ export function importGroups(formData: FormData): Promise<Response> {
   });
 }
 
-export function getRecentJobs(): Promise<Response> {
-  return fetch(`/api/v1/job/`);
```
Comment (syucream) [inline frontend/src/utils/AironeAPIClient.ts:282]: We can use API V2 :)

File: job/api_v2/views.py [hunk 2]
```diff
@@ -39,4 +47,7 @@ def get_queryset(self):
             ),
         )
 
+        if created_after:
+            query &= Q(created_at__gte=created_after)
```
Comment (syucream) [inline job/api_v2/views.py:51]: To support filtering recent jobs.

**Review by hinashi — APPROVED** [review]
LGTM!

**Review by userlocalhost — APPROVED** [review]
Thank you for excellent work!! 👍

**Pull request merged**